test/e2e/performanceprofile/functests/utils/mcps/mcps.go

```diff
@@ -296,3 +299,29 @@ func Delete(name string) error {
 	}
 	return WaitForDeletion(context.TODO(), client.ObjectKey{Name: name}, 2*time.Minute)
 }
+
+func EnableCrun(name string, perfprofile *performancev2.PerformanceProfile, profileMCP *machineconfigv1.MachineConfigPool) *machineconfigv1.ContainerRuntimeConfig {
+	ctrcfg1 := &machineconfigv1.ContainerRuntimeConfig{
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+		Spec: machineconfigv1.ContainerRuntimeConfigSpec{
+			MachineConfigPoolSelector: &metav1.LabelSelector{
+				MatchLabels: profile.GetMachineConfigPoolSelector(perfprofile, profileMCP),
+			},
+			ContainerRuntimeConfig: &machineconfigv1.ContainerRuntimeConfiguration{
+				DefaultRuntime: machineconfigv1.ContainerRuntimeDefaultRuntimeCrun,
+			},
+		},
+	}
+	return ctrcfg1
+}
+
+func GetValueFromCrioConfig(workercnfNode *corev1.Node, key, confPath string) ([]string, error) {
+	out, err := nodes.ExecCommandOnMachineConfigDaemon(workercnfNode, []string{"cat", filepath.Join("/rootfs", confPath)})
+	if err != nil {
+		msg := fmt.Sprintf("failed to fetch values from %s", confPath)
```

Comment:
you can simplify:
`return fmt.Errorf("failed to fetch values from %s: %w", confPath, err)`

Reply:
Addressed in the latest commit